Yes, it seems the ARM build fails on Windows. Interestingly it's not in the blend2d code that the failure occurs, but rather in asmjit.

```
FAILED: CMakeFiles/blend2d.dir/3rdparty/asmjit/src/asmjit/arm/a64builder.cpp.obj 
C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1434~1.319\bin\Hostx64\arm64\cl.exe   /TP  -ID:\buildtrees\blend2d\src\28ab5c6690-f673a4e089.clean\3rdparty\asmjit\src /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP  /D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1  -MP -GR- -GF -Zc:__cplusplus -Zc:inline -Zc:strictStrings -Zc:threadSafeInit- -W4 -DASMJIT_STATIC -DASMJIT_NO_STDCXX -GS /showIncludes /FoCMakeFiles\blend2d.dir\3rdparty\asmjit\src\asmjit\arm\a64builder.cpp.obj /FdCMakeFiles\blend2d.dir\ /FS -c D:\buildtrees\blend2d\src\28ab5c6690-f673a4e089.clean\3rdparty\asmjit\src\asmjit\arm\a64builder.cpp
cl : Command line warning D9025 : overriding '/GR' with '/GR-'
cl : Command line warning D9025 : overriding '/W3' with '/W4'
D:\buildtrees\blend2d\src\28ab5c6690-f673a4e089.clean\3rdparty\asmjit\src\asmjit\arm\../arm/a64emitter.h(218): warning C4002: too many arguments for function-like macro invocation 'mvn'
D:\buildtrees\blend2d\src\28ab5c6690-f673a4e089.clean\3rdparty\asmjit\src\asmjit\arm\../arm/a64emitter.h(219): warning C4002: too many arguments for function-like macro invocation 'mvn'
D:\buildtrees\blend2d\src\28ab5c6690-f673a4e089.clean\3rdparty\asmjit\src\asmjit\arm\../arm/a64emitter.h(866): warning C4002: too many arguments for function-like macro invocation 'mvn'
D:\buildtrees\blend2d\src\28ab5c6690-f673a4e089.clean\3rdparty\asmjit\src\asmjit\arm\../arm/a64emitter.h(219): error C2535: 'asmjit::_abi_1_9::Error asmjit::_abi_1_9::a64::EmitterExplicitT<This>::neon_not(const asmjit::_abi_1_9::arm::Gp &)': member function already defined or declared
D:\buildtrees\blend2d\src\28ab5c6690-f673a4e089.clean\3rdparty\asmjit\src\asmjit\arm\../arm/a64emitter.h(218): note: see declaration of 'asmjit::_abi_1_9::a64::EmitterExplicitT<This>::neon_not'
D:\buildtrees\blend2d\src\28ab5c6690-f673a4e089.clean\3rdparty\asmjit\src\asmjit\arm\../arm/a64emitter.h(1204): note: see reference to class template instantiation 'asmjit::_abi_1_9::a64::EmitterExplicitT<This>' being compiled
D:\buildtrees\blend2d\src\28ab5c6690-f673a4e089.clean\3rdparty\asmjit\src\asmjit\arm\../arm/a64emitter.h(219): error C2535: 'asmjit::_abi_1_9::Error asmjit::_abi_1_9::a64::EmitterExplicitT<asmjit::_abi_1_9::a64::Emitter>::neon_not(const asmjit::_abi_1_9::arm::Gp &)': member function already defined or declared
D:\buildtrees\blend2d\src\28ab5c6690-f673a4e089.clean\3rdparty\asmjit\src\asmjit\arm\../arm/a64emitter.h(218): note: see declaration of 'asmjit::_abi_1_9::a64::EmitterExplicitT<asmjit::_abi_1_9::a64::Emitter>::neon_not'
D:\buildtrees\blend2d\src\28ab5c6690-f673a4e089.clean\3rdparty\asmjit\src\asmjit\arm\../arm/a64emitter.h(1211): note: see reference to class template instantiation 'asmjit::_abi_1_9::a64::EmitterExplicitT<asmjit::_abi_1_9::a64::Emitter>' being compiled
D:\buildtrees\blend2d\src\28ab5c6690-f673a4e089.clean\3rdparty\asmjit\src\asmjit\arm\../arm/a64emitter.h(219): error C2535: 'asmjit::_abi_1_9::Error asmjit::_abi_1_9::a64::EmitterExplicitT<asmjit::_abi_1_9::a64::Assembler>::neon_not(const asmjit::_abi_1_9::arm::Gp &)': member function already defined or declared
D:\buildtrees\blend2d\src\28ab5c6690-f673a4e089.clean\3rdparty\asmjit\src\asmjit\arm\../arm/a64emitter.h(218): note: see declaration of 'asmjit::_abi_1_9::a64::EmitterExplicitT<asmjit::_abi_1_9::a64::Assembler>::neon_not'
D:\buildtrees\blend2d\src\28ab5c6690-f673a4e089.clean\3rdparty\asmjit\src\asmjit\arm\../arm/a64assembler.h(21): note: see reference to class template instantiation 'asmjit::_abi_1_9::a64::EmitterExplicitT<asmjit::_abi_1_9::a64::Assembler>' being compiled
D:\buildtrees\blend2d\src\28ab5c6690-f673a4e089.clean\3rdparty\asmjit\src\asmjit\arm\../arm/a64emitter.h(219): error C2535: 'asmjit::_abi_1_9::Error asmjit::_abi_1_9::a64::EmitterExplicitT<asmjit::_abi_1_9::a64::Builder>::neon_not(const asmjit::_abi_1_9::arm::Gp &)': member function already defined or declared
D:\buildtrees\blend2d\src\28ab5c6690-f673a4e089.clean\3rdparty\asmjit\src\asmjit\arm\../arm/a64emitter.h(218): note: see declaration of 'asmjit::_abi_1_9::a64::EmitterExplicitT<asmjit::_abi_1_9::a64::Builder>::neon_not'
D:\buildtrees\blend2d\src\28ab5c6690-f673a4e089.clean\3rdparty\asmjit\src\asmjit\arm\../arm/a64builder.h(23): note: see reference to class template instantiation 'asmjit::_abi_1_9::a64::EmitterExplicitT<asmjit::_abi_1_9::a64::Builder>' being compiled
```

I briefly inspected the relevant asmjit source code, and the function-like macro "mvn" it's complaining about doesn't look like it's actually a macro. It appears to be just an identifier provided as an argument to a macro, which then gets used as a function name. I wonder if this is an MSVC bug.

[install-arm64-windows-dbg-out.log](https://github.com/microsoft/vcpkg/files/10515399/install-arm64-windows-dbg-out.log)

--- a/ports/blend2d/vcpkg.json
+++ b/ports/blend2d/vcpkg.json
@@ -1,11 +1,12 @@
 {
   "name": "blend2d",
   "version-date": "2022-12-31",
+  "port-version": 1,
   "description": "Beta 2D Vector Graphics Powered by a JIT Compiler",
   "homepage": "https://github.com/blend2d/blend2d",
   "documentation": "https://blend2d.com/doc/index.html",
   "license": "Zlib",
-  "supports": "!arm & !uwp & !wasm32",
+  "supports": "!(arm & windows) & !uwp & !wasm32",
   "dependencies": [
     {
       "name": "vcpkg-cmake",

--- a/ports/osgearth/vcpkg.json
+++ b/ports/osgearth/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "osgearth",
   "version": "3.3",
-  "port-version": 4,
+  "port-version": 5,
   "description": "osgEarth - Dynamic map generation toolkit for OpenSceneGraph Copyright 2021 Pelican Mapping.",
   "homepage": "https://github.com/gwaldron/osgearth",
   "license": "LGPL-3.0-or-later",
@@ -54,7 +54,7 @@
           "features": [
             "blend2d"
           ],
-          "platform": "!arm & !uwp & !wasm32"
+          "platform": "!(arm & windows) & !uwp & !wasm32"
         }
       ]
     },

--- a/versions/b-/blend2d.json
+++ b/versions/b-/blend2d.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "dc33e610ee62c7cc13415a1e18e682ccbb56a68d",
+      "version-date": "2022-12-31",
+      "port-version": 1
+    },
+    {
       "git-tree": "838a7227b29f1b1beb7daf181cfc97e5674e0270",
       "version-date": "2022-12-31",
       "port-version": 0

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5738,7 +5738,7 @@
     },
     "osgearth": {
       "baseline": "3.3",
-      "port-version": 4
+      "port-version": 5
     },
     "osi": {
       "baseline": "0.108.6",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -550,7 +550,7 @@
     },
     "blend2d": {
       "baseline": "2022-12-31",
-      "port-version": 0
+      "port-version": 1
     },
     "blitz": {
       "baseline": "2020-03-25",

--- a/versions/o-/osgearth.json
+++ b/versions/o-/osgearth.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "606bbaf1456dc9d5a6f5ef879a54b894c4ace5b3",
+      "version": "3.3",
+      "port-version": 5
+    },
+    {
       "git-tree": "208b0c23a8e79910067bb0b8a6dc589da7307de5",
       "version": "3.3",
       "port-version": 4

